docs/how-to-customize/index.md

```diff
@@ -0,0 +1,25 @@
+# How to Customize Atomist
```

Comment:
Are people going to appreciate the difference between configuring and customizing?

Reply:
It's a good point, and a clear one as a Spring-person but it might be confusing... Let me give this one some more thought.